documentation/Color system.md

```diff
@@ -274,3 +274,4 @@ Used to decorate elements where color does convey a specific meaning in componen
 | `--p-range-slider-thumb-size-active`      | `2.4rem`                                                                                                     |
 | `--p-range-slider-thumb-scale`            | `1.5`                                                                                                        |
 | `--p-badge-font-weight`                   | `500`                                                                                                        |
+| `--p-frame-background`                    | `rgb(244, 245, 246)`                                                                                         |
```

Comment:
This is temporary until we can make the background change in tokens: https://github.com/Shopify/polaris-tokens/blob/master/src/configs/base.ts

Reply:
Should we make a PR in tokens to kick off this change?

Reply:
This new color is very similar to the old background color but it's slightly darker. I'll look into adding it to tokens but I'm not sure how to reverse engineer a hex color into hue, saturation, lightness values. Might take some digging

Reply:
I forgot chrome gives you hsl values so I should be able to base it off: `hsl(210deg, 10%, 96%)`. I'll put up a pr shortly

Reply:
Yeah you can I think `ctrl` click the color and it will convert between the different values 💯